jobs/JGDAS_GLOBAL_OCEAN_ANALYSIS_BMAT_VRFY

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+export STRICT="NO"
+source "${HOMEgfs}/ush/preamble.sh"
+
+
+export DATA="${DATAROOT}/${RUN}ocnanal_${cyc}"
```

Comment:
Is there a reason why every `DATA` is `${RUN}ocnanl_${cyc}`?
We need this solely if it will need to be preserved after this job is completed.
If it is not needed, then we should stick with the default `${job}.${jobid}` that are defined upstream.

This comment applies to the `JGDAS_GLOBAL_OCEAN_ANALYSIS_VRFY` as well.

Reply:
yes, at least for now it needs to be preserved.

Reply:
This directory is being gutted here:
https://github.com/NOAA-EMC/global-workflow/blob/f98433ff3661ef848d381ae65322623573a114cc/jobs/JGDAS_GLOBAL_OCEAN_ANALYSIS_POST#L21

I suggest the post job to copy what it needs to `COM` and create a working directory for this job in `$DATAROOT`

Reply:
ok, will look into it @aerorahul

Reply:
To elaborate a bit further: `$DATA` should only contain temporary data during runtime and is wiped at the end of the job. Typically that means a random directory name is okay. With the prep/run/post paradigm, that doesn't work because RUN needs to know where PREP staged stuff, but prep/run/post should still be considered a single job for purposes of `$DATA` and deleted at the end of POST. Anything that is needed outside of that "one" job needs to get placed into `$COM`.

Reply:
Understood @WalterKolczynski-NOAA , I'll push the modifications that address that issue (#1273 ) later today ... maybe. 
I'll use the default `$DATA` (default is `${job}.${jobid}`?) for the `JGDAS_GLOBAL_OCEAN_ANALYSIS_VRFY`, that job will use data saved in `$COM`. The rest of the marine DA j-jobs all depend on PREP, so I'll have to stick to the specified `$DATA=${RUN}ocnanal_${cyc}`.